pandas/tests/util/test_deprecate_nonkeyword_arguments.py

```diff
@@ -155,3 +165,17 @@ def test_class():
     )
     with tm.assert_produces_warning(FutureWarning, match=msg):
         Foo().baz("qux", "quox")
+
+
+def test_deprecated_keywords():
+    msg = (
+        "In the future version of pandas the arguments args"
+        "and kwargs will be deprecated for these functions"
+    )
+    with tm.assert_produces_warning(FutureWarning, match=msg):
+        assert cummax
```

Comment:
`assert [statement]` evaluates the truthiness of the statement. For example, `[]` will evaluate to False (and hence raise if used in an assert) whereas `[1]` will evaluate to True. Functions are always truthy - they evaluate to True.

Instead, this needs to call the function itself, e.g.

```
df = DataFrame({'a': [1, 1, 2], 'b': [3, 4, 5]})
gb = df.groupby('a')
gb.cummax(kwargs=1)
```

Reply:
Oh thanks

Reply:
i did the changes but still the tests are failing, don't know why